### DISCUSSION TITLE
improvement?

### DISCUSSION THREAD

**Discussion by hazzarr1** [Anarios/return-youtube-dislike#689] [Ideas]
hi, just had an idea
if the youtube api stops returning dislikes, it would be possible to create a DB where actual dislikes are stored along with the video id instead of estimating. and then overlay the dislike button with with a button that links directly to external DB.

the hosting would cost a bit but could ask for donations from creators who want to bring back the dislike button.

**Comment by cyrildtm** [comment]
it's already using a separate DB. 
The "estimation" is about extrapolating.

**Reply by hazzarr1** [reply]
damn it ok
no worries then
